resources/install.rb

```diff
@@ -0,0 +1,60 @@
+# resources/install.rb
+
+# Resource for InfluxDB install
+
+property :arch_type, kind_of: String
+property :include_repository, kind_of: [TrueClass, FalseClass], default: true
+property :influxdb_key, kind_of: String, default: 'https://repos.influxdata.com/influxdb.key'
+default_action :install
+
+include InfluxdbCookbook::Helpers
+
+action :install do
+  if node.platform_family? 'rhel'
+    yum_repository 'influxdb' do
+      description 'InfluxDB Repository - RHEL \$releasever'
+      baseurl node['influxdb']['upstream_repository']
+      gpgkey influxdb_key
+      only_if { include_repository }
+    end
+  elsif node.platform_family? 'debian'
+    # see if we should auto detect
+    unless new_resource.arch_type
+      new_resource.arch_type determine_arch_type(new_resource, node)
+    end
+    apt_repository 'influxdb' do
```

Comment:
Sorry for the confusion.  Upon inspection of code, it appeared that the package wasnt needed and you had confirmed it.  However, I have been checking builds based off docker images.  From clean slate on ubuntu distros, chef converges fail.  Its complaining about not having access to https.

```
  * execute[apt-get update] action run[2016-05-06T21:22:22+00:00] ERROR: execute[apt-get update] (/tmp/kitchen/cache/cookbooks/apt/providers/repository.rb line 196) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '100'
       ---- Begin output of apt-get update -o Dir::Etc::sourcelist='sources.list.d/influxdb.list' -o Dir::Etc::sourceparts='-' -o APT::Get::List-Cleanup='0' ----
       STDOUT:
       STDERR: E: The method driver /usr/lib/apt/methods/https could not be found.
```

If its not too much to ask, can you please add it back?